.tool-versions

```diff
@@ -1,6 +1,6 @@
   # This file is for you! Please, updated to the versions agreed by your team.
 
-terraform 1.7.0
```

Comment:
I think going forwards, it would be better to not manage terraform via asdf, tfenv is way better and scaffold also uses that to manage versions so would be consistent.

Reply:
updated